hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -835,6 +842,26 @@ private void initializeExchangeRateManager(@NonNull final HederaState state) {
         logger.info("Exchange rates initialized");
     }
 
+    private void initializeThrottleManager(@NonNull final HederaState state) {
+        logger.info("Initializing throttles");
+        final var readableFileStore = new ReadableStoreFactory(state).getStore(ReadableFileStore.class);
```

Comment:
There is a lot of code duplicated. Maybe we can extract the common functionality from these initialize...-methods.

Reply:
Will fix it in https://github.com/hashgraph/hedera-services/pull/8147